extension/zpagesextension/README.md

```diff
@@ -27,13 +27,17 @@ The following settings are required:
 zPages. Use localhost:<port> to make it available only locally, or ":<port>" to
 make it available on all network interfaces.
 
+The following settings can be optionally configured:
+
+- `enable_expvar` (default = false): Enable the expvar services. For detail see [ExpvarZ](#expvarz).
+
 Example:
 ```yaml
 extensions:
   zpages:
 ```
 
-The full list of settings exposed for this exporter are documented [here](./config.go)
+The full list of settings exposed for this extension are documented [here](./config.go)
```

Comment:
Separate PR for this

Reply:
uncalled for. Who would go through pain of new PR for a typo?

Reply:
Pain? What pain?

This change is not related to the PR. It is best done in a separate PR, so we have a clear git history and if we need to revert this PR, we don't revert this change as well.

I am happy to open that PR.

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/12119

Reply:
Got it, I agree with the purpose of the possibility of revert, I'll commit the suggestion.

Reply:
> Pain? What pain?

Conservatively, it takes at least 1hr of back and forth for external contributor to complete a PR in this repo. I would never come back to do it just to fix an obvious typo.

Reply:
OK, let's go about this with a bit of method.
When I say that a typo belongs in a separate PR, I am saying that it doesn't belong in this PR. I invite you to open a new PR but *you can say no* and *you don't have to do it*. Free agency applies, right?

All I am saying is that the typo needs to be dealt with separately. Sorry if I'm being too sparse in my feedback - happy to add more to our best practices to cover this more eloquently.

There are a few things you can do:
* You can open an issue based on my comment with a few clicks.
* You can also open the github web interface and make the change without leaving the web browser.

Admittedly my pain threshold must be higher than external contributors, so this might not hurt me as much.

Reply:
> let's go about this with a bit of method.

let's not. We already spent 10x mental energy discussing this. The typo is not fixed, may or may not get fixed. Your higher "pain threshold" is not a good reason to force external contributors to do additional and imo pointless work (requiring a PR/changelog for a non-functional typo is a negative ROI).

Reply:
Well I did open another PR to take care of it, so it's likely to get fixed, maybe you'd like to review? https://github.com/open-telemetry/opentelemetry-collector/pull/12119